mmdeploy/codebase/mmaction/models/recognizers/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+from .base import base_recognizer__forward
```

Comment:
you can just use `from . import xxx` since we do not need to expose these rewriters